test_files/import_alias/importer.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * @fileoverview added by tsickle
+ * @suppress {checkTypes} checked by tsc
+ */
+goog.module('test_files.import_alias.importer');
+var module = module || { id: 'test_files/import_alias/importer.ts' };
+var exporter_1 = goog.require('test_files.import_alias.exporter');
+const tsickle_forward_declare_1 = goog.forwardDeclare("test_files.import_alias.exporter");
```

Comment:
Odd that the forwardDeclare is emitted even when not used. Am I right that it doesn't affect the closure check that you are fixing here?

Reply:
It is maybe somewhat odd, but does simplify of code a bit – we just unconditionally forward declare any import, and then always use the forward declare (not the regular import) in all type declarations. That's consistent, and does not conflict with the require, and also doesn't trigger any of the Closure linter warnings that often cause issues with requires.